core/templates/dev/head/pages/library/Library.js

```diff
@@ -80,6 +80,31 @@ oppia.controller('Library', [
       $http.get('/libraryindexhandler').success(function(data) {
         $scope.libraryGroups = data.activity_summary_dicts_by_category;
 
+        $http.get('/creatordashboardhandler/data')
+          .then(function(response) {
+            if (response.data.explorations_list !== undefined) {
```

Comment:
Why would it be undefined? If it's because the user is logged out then why not avoid making the call in the first place if that's the case?

Reply:
okay.